### PR TITLE
Add nuget.config to fix compilation

### PR DESCRIPTION
**Pull request by Beyley**
# Summary of the PR
Fix compilation

# Related issues, Discord discussions, or proposals
https://canary.discord.com/channels/521092042781229087/587346162802229298/1007441794394488954

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<configuration>
+	<packageSources>
+		<clear />
+  		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+	</packageSources>
+	<packageSourceMapping>
+  		<packageSource key="nuget.org">
+    			<package pattern="*" />
+  		</packageSource>
+	</packageSourceMapping>
```
Comment (Perksey) [inline nuget.config:11]: identation very strange

Comment (Redhacker1) [reply nuget.config:11]: Well when you copy paste from discord to an IDE that is what happens

Comment (Beyley) [reply nuget.config:11]: idk its whatever neovim decided, ill reformat it to 2 spaces which seems to be the standard for XML?

Comment (Beyley) [reply nuget.config:11]: > Well when you copy paste from discord to an IDE that is what happens

that was not the issue

**Review by Perksey — APPROVED** [review]
Not a maintainer but I'm sure there'll be no objections in me approving something this simple ;)

**Pull request merged**